site/docs/v1/tech/apis/users.adoc

```diff
@@ -880,6 +880,14 @@ include::docs/v1/tech/apis/_standard-put-response-codes.adoc[]
 The email verification Id that was generated by this API request. This identifier may be used by the <<verify-a-users-email,Verify a User's Email>> API.
 This field is only returned in the JSON response body if the request was authenticated using an API key, if an API key is not used no response body is returned.
 
+[field]#verificationOneTimeCode# [type]#[Integer]#::
```

Comment:
Is this always an integer? I thought it could be something else? The advanced tab of the tenant config indicates it can be alphanumeric?

Reply:
Agree -  I have updated to "string" which I believe to be accurate.